src/js/pages/universe/PackagesTab.js

```diff
@@ -30,6 +33,18 @@ const METHODS_TO_BIND = [
   'handleSearchStringChange'
 ];
 
+const shouldRenderUniverseOption = Hooks.applyFilter(
+  'hasCapability', false, 'packageAPI'
+);
+
+if (shouldRenderUniverseOption) {
```

Comment:
Can you expand on how this could be accomplished? This option should be available to all Open DC/OS users, so it can't be a added via a plugin (I implemented it in a plugin by mistake initially; this PR is fixing that mistake).

@rcorral and I discussed this and decided `PackagesTab` was the best place to handle this.